# Enable detection of system lzma library as header is in /usr/include
diff -urNad mysql-8.0.4-rc-orig/cmake/lzma.cmake mysql-8.0.4-rc/cmake/lzma.cmake
--- mysql-8.0.4-rc-orig/cmake/lzma.cmake	2018-01-18 15:52:52.000000000 -0000
+++ mysql-8.0.4-rc/cmake/lzma.cmake	2018-01-24 09:47:41.264835871 -0000
@@ -24,7 +24,7 @@
 # bundled is the default
 
 MACRO (FIND_SYSTEM_LZMA)
-  FIND_PATH(PATH_TO_LZMA NAMES lzma/lzma.h)
+  FIND_PATH(PATH_TO_LZMA NAMES lzma.h)
   FIND_LIBRARY(LZMA_SYSTEM_LIBRARY NAMES lzma)
   IF (PATH_TO_LZMA AND LZMA_SYSTEM_LIBRARY)
     SET(SYSTEM_LZMA_FOUND 1)
